specification/schemas/return-orders/DemoOrderResponse.json

```diff
@@ -0,0 +1,51 @@
+{
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/DemoOrder"
+    },
+    {
+      "required": [
+        "id",
+        "attributes",
+        "relationships"
```

Comment:
same here, relationships is set as required but definition is missing.

Reply:
I believe adding the `shop` relationship on the DemoOrder solves both cases of missing relationships, right? Since DemoOrderResponse uses `allOf: [ { "$ref": ".../schemaPath/DemoOrder.json"  } ]`